metis-performance-metric/src/main/java/eu/europeana/metis/performance/metric/PerformanceMetricMain.java

```diff
@@ -26,23 +23,31 @@ public class PerformanceMetricMain {
     private static final DateTimeFormatter ISO_FORMATTER = DateTimeFormatter.ISO_DATE_TIME;
     private static final LocalDateTime NOW_DATE = LocalDateTime.now();
     private static final String FILE_PATH = "metis-performance-metric/src/main/java/eu/europeana/metis/performance/metric/output/";
-    private static final String OUTPUT_FILE_NAME = "performance-metric-2-";
+    private static final String PERFORMANCE_METRIC_1 = "performance-metric-1-";
+    private static final String PERFORMANCE_METRIC_2 = "performance-metric-2-";
 
-    public static void main(String[] args) throws CustomTruststoreAppender.TrustStoreConfigurationException, FileNotFoundException, ParseException {
+    public static void main(String[] args) throws CustomTruststoreAppender.TrustStoreConfigurationException {
         LOGGER.info("Starting script");
 
         final MongoMetisCoreDao mongoMetisCoreDao = new MongoMetisCoreDao(propertiesHolder);
-        final CSVUtilities csvUtilities = new CSVUtilities(mongoMetisCoreDao);
+        final PerformanceMetricsUtilities performanceMetricsUtilities = new PerformanceMetricsUtilities(mongoMetisCoreDao);
+        final CSVUtilities csvUtilities = new CSVUtilities(performanceMetricsUtilities);
 
         //Prepare input values
         final String startDateAsString = "2023-01-01 00:00:00";
-        final String endDateAsString = "2023-04-26 23:59:59";
-        final SimpleDateFormat simpleDateFormatter = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
-        final Date startDate = simpleDateFormatter.parse(startDateAsString);
-        final Date endDate = simpleDateFormatter.parse(endDateAsString);
-
-        //Start printing
-        csvUtilities.writeIntoCsvFile(FILE_PATH + OUTPUT_FILE_NAME + NOW_DATE.format(ISO_FORMATTER) + ".csv", startDate, endDate);
+        final String endDateAsString = "2023-05-01 23:59:59";
+        final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+        final LocalDateTime startDate = LocalDateTime.parse(startDateAsString, formatter);
+        final LocalDateTime endDate = LocalDateTime.parse(endDateAsString, formatter);
+
+        //Start printing metrics
+        LOGGER.info("Running metrics 1 script");
+        csvUtilities.writeMetric1IntoCsvFile(FILE_PATH + PERFORMANCE_METRIC_1 + NOW_DATE.format(ISO_FORMATTER) + ".csv", startDate, endDate);
```

Comment:
for the filenames, I suggest to change `NOW_DATE.format(ISO_FORMATTER)` to a more generic format file name in the sense that in Linux filenames can contain special symbols but in Windows this doesn't work. 
```
performance-metric-2-2023-05-11T13:18:21.508332
```
to
```
performance-metric-2-2023-05-11T13_18_21_508332
```